README.md

```diff
@@ -69,29 +69,22 @@ my portfolio is my first microverse project and will be my professional portfoli
 <details>
   <summary>Client</summary>
   <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
+    <li>index.html</li>
   </ul>
 </details>
 
 <details>
   <summary>Server</summary>
   <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
-  </ul>
-</details>
-
-<details>
-<summary>Database</summary>
-  <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+    <li>style.css</li>
   </ul>
```

Comment:
[OPTIONAL] Note that your project does not have a backend/server so I suggest you indicate that you did not have any instead of style.css 😉